prow/github/client.go

```diff
@@ -268,3 +268,25 @@ func (c *Client) CloseIssue(owner, repo string, number int) error {
 	}
 	return nil
 }
+
+// FindIssues uses the github search API to find issues which match a particular query.
```

Comment:
Add a TODO saying that we should accept `map[string][]string` and use `net/url` properly.